mockwebserver/src/main/java/okhttp3/internal/SslContextBuilder.java

```diff
@@ -38,10 +41,14 @@
  * instances where possible.
  */
 public final class SslContextBuilder {
```

Comment:
. . . .with this change I think this isn’t building an SSLContext, it’s instead building like, a `SSLContext` and a `TrustManager[]` array. Would it be offensive to come up with a new word to represent that pair of things? Maybe like `SslClient` and `SslClient.Builder` ? The `SslClient` is the immutable thing with two fields (sslContext, trustmanagers) and the `SslClient.Builder` is its builder?